### PR TITLE
Feat(status): [450] tweet status

### PR DESCRIPTION
**Pull request by ilmiawan**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
add link to post current status notification body to twitter

## How did you implement / how did you fix it  
1. create tweet message of current status
2. add message to twitter text link
3. add to status notification body

## How to test  
1. create monika config with email notification
2. run monika with `monika --status-notification "*/1 * * * *"`
3. check on mail status notification, there should be a link to twitter.

### CODE REVIEW

**Review by gsukanto — APPROVED** [review]

File: src/jobs/summary-notification.ts [hunk 1]
```diff
@@ -226,3 +236,29 @@ export async function printSummary(cliConfig: IConfig) {
...
+}): string {
+  const message = `I'm using Monika by @hyperjump_tech to monitor ${number_of_probes} probes! In the last 24 hours, 
+
+⏱ the average response time is ${average_response_time} ms
+⚠️ there were ${number_of_incidents} incidents
+✅ and ${number_of_recoveries} recoveries!
+
+Give it a try!
+
+opensource
```
Comment (dennypradipta) [inline src/jobs/summary-notification.ts:259]: Is this supposed to be a hashtag or just plain text?

Comment (dennypradipta) [inline src/jobs/summary-notification.ts:259]: Also if you want to use hashtags, use #monika instead for better discoverability.

Comment (nicnocquee) [reply src/jobs/summary-notification.ts:259]: use both `opensource` and `monika` hashtags.

**Review by nicnocquee — CHANGES_REQUESTED** [review]

**Review by sapiderman — APPROVED** [review]
pending feedback fixes from denny + nico this looks ok

File: src/jobs/summary-notification.ts (hunk 1 above)
Comment (ilmiawan) [reply src/jobs/summary-notification.ts]: fixed by adding opensource to hashtags

Comment (ilmiawan) [reply src/jobs/summary-notification.ts]: fixed by adding opensource and monika to hashtags

**Review by dennypradipta — APPROVED** [review]
Well done!
image

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**